android/src/main/java/com/rnfingerprint/FingerprintHandler.java

```diff
@@ -0,0 +1,83 @@
+package com.rnfingerprint;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.hardware.fingerprint.FingerprintManager;
+import android.os.CancellationSignal;
+import android.support.v4.app.ActivityCompat;
+import android.widget.Toast;
+
+public class FingerprintHandler extends FingerprintManager.AuthenticationCallback {
+
+    private CancellationSignal cancellationSignal;
+    private boolean selfCancelled;
+    private Context appContext;
+
+    private final FingerprintManager mFingerprintManager;
+    private final Callback mCallback;
+
+    public FingerprintHandler(Context context, FingerprintManager fingerprintManager, Callback callback) {
+        appContext = context;
```

Comment:
It keep names consistent, so if we use `mCallback` shouldn't we also use `mAppContext` ?